main.rb

```diff
@@ -0,0 +1,173 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+    choices
+  end
+
+  def choices
+    puts
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    user_choice = gets.chomp
+    start_options user_choice
+  end
+
+  def start_options(options)
+    case options
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rental_by_person_id
+    else
+      puts 'Exit'
+    end
+  end
```

Comment:
- [ ] Only this code that serves as the entry point and the code in the last lines of this file for starting the app should be in this `main.rb` file